src/main/java/com/example/employeeTDD/demo/Controller/EmployeeController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.employeeTDD.demo.Controller;
+
+import com.example.employeeTDD.demo.Entity.Employee;
+import com.example.employeeTDD.demo.Exception.EmployeeNotFoundException;
+import com.example.employeeTDD.demo.ServiceTest.EmployeeService;
+import org.apache.velocity.exception.ResourceNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/employee")
+public class EmployeeController {
+
+    private EmployeeService employeeService;
+
+    @Autowired
+    public void setEmployeeService(EmployeeService employeeService) { this.employeeService = employeeService; }
+
+    @PostMapping("/create")
```

Comment:
When you create endpoints related to same resource, you should have the noun as the mapping with GET and POST differentiating what they do. For eg if we create employees, the requests should be

/employee (POST) to create
/employee/{id} (PUT) to update
/employee/{id} (GET) to get an employee
/employees (GET) list all employees
/employee/{id} (DELETE) delete an employee

Reply:
acknowledged - 8aea984